docs/DeveloperGuide.md

```diff
@@ -248,35 +251,88 @@ is better than these alternatives.<br>
 a. Include those libraries in the repo (this bloats the repo size)<br>
 b. Require developers to download those libraries manually (this creates extra work for developers)<br>
 
+
 ## Appendix A : User Stories
 
 Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (unlikely to have) - `*`
 
-
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
-`* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
+`* * *` | New user | See usage instructions | Refer to instructions when I forget how to use the App
+`* * *` | User | Add a task | Keep track of tasks which I need to do
+`* * *` | User | Delete a task/multiple tasks | Remove tasks that I no longer need to keep track of
+`* * *` | User | Edit a task name | Update task details to reflect the latest changes
+`* * *` | User | View all my tasks | Have a quick and clear reference of everything I need to do
+`* * *` | User | Mark a task/multiple tasks as completed | Know that it is completed without deleting it, distinguish between completed and uncompleted tasks
+`* * *` | User | Unmark a task from completed | Update the status of my task
+`* * *` | User | Undo my last action(s) | Easily correct any accidental mistakes in the last command(s)
+`* * *` | User | Search based on task name | Find a task without going through the entire list if I remember a few key words
```

Comment:
I see a lot of high priority features. Hope you have scheduled them with realistic deadlines.